dces-drc-integration/src/main/java/uk/gov/justice/laa/crime/dces/integration/controller/TempTestController.java

```diff
@@ -42,6 +54,14 @@ public class TempTestController {
     private final StubValidation stubValidation;
     private final ObjectMapper objectMapper;
     private final DrcClient drcClient;
+    private static final int REQUEST_ID_LIST_SIZE_LIMIT_CONCOR = 350;
+    private static final int REQUEST_ID_LIST_SIZE_LIMIT_FDC = 1000;
+
+    @Autowired
+    ContributionService concorContributionsService;
```

Comment:
`@Autowired` should only really be used in tests. For production code, it's usually better to use constructor injection or setter injection.

For example, `DrcClient` is currently set by constructor injection (because of `@RequiredArgsConstructor` and because the field is `private final` so the Lombok-generated constructor includes it).

If you make these two fields (`concorContributionService` and `fdcService`) be `private final` fields too, then they will automatically be injected by Spring into the Lombok-generated constructor (which will include these fields), and then `@Autowired` is not needed.

Reply:
Good spot and great explanation! Done.